ios/ShareViewController.swift

```diff
@@ -44,7 +44,12 @@ class ShareViewController: SLComposeServiceViewController {
         return
       }
 
-      handlePost(item)
+      if (contentText != nil && contentText != "") {
+        let extraData: [String: Any] = ["userInput": contentText as String]
+        handlePost(item, extraData)
```

Comment:
Do we need to write it like this to work with Swift 5 ?

```suggestion
        handlePost(item, extraData: extraData)
```